modules/xresources/hm.nix

```
@@ -1,40 +1,44 @@
-{ config, lib, ... }:
+{ mkTarget, ... }:
+mkTarget {
+  name = "xresources";
+  humanName = "Xresources";
 
-{
-  options.stylix.targets.xresources.enable =
-    config.lib.stylix.mkEnableTarget "Xresources" true;
-
-  config =
-    lib.mkIf (config.stylix.enable && config.stylix.targets.xresources.enable)
+  configElements = [
+    (
+      { fonts }:
+      {
+        xresources.properties = {
+          "*.renderFont" = true;
+          "*.faceName" = fonts.monospace.name;
+          "*.faceSize" = toString fonts.sizes.terminal;
```

This here was previously sorted as:

```suggestion
          "*.faceName" = fonts.monospace.name;
          "*.faceSize" = toString fonts.sizes.terminal;
          "*.renderFont" = true;
```